<launch>
  <arg name="lidar_detections_topic" default="/detection/lidar_detector/objects"/>
  <arg name="lidar_tracklets_topic" default="/detection/lidar_ground_truth/objects"/>
  <arg name="overlap_threshold" default="0.6"/>
  <arg name="sync_topics" default="true"/>

  <node name="lidar_tracklet_compare_01" pkg="lidar_tracklet_compare" type="lidar_tracklet_compare" output="screen">
    <param name="lidar_detections_topic" value="$(arg lidar_detections_topic)"/>
    <param name="lidar_tracklets_topic" value="$(arg lidar_tracklets_topic)"/>
    <param name="overlap_threshold" value="$(arg overlap_threshold)"/>
    <param name="sync_topics" value="$(arg sync_topics)"/>
  </node>

</launch>
